src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/configuration/SubscribeTarget.java

```diff
@@ -16,19 +16,25 @@
 
 package com.exactpro.th2.common.schema.message.impl.rabbitmq.configuration;
 
-import lombok.Builder;
 import lombok.EqualsAndHashCode;
-import lombok.Getter;
 import lombok.ToString;
 
-@Getter
-@Builder
 @ToString
 @EqualsAndHashCode
 public class SubscribeTarget {
-
     private String queue;
-
     private String routingKey;
 
+    public SubscribeTarget(String queue, String routingKey) {
+        this.queue = queue;
+        this.routingKey = routingKey;
+    }
+
+    public String getQueue() {
+        return queue;
+    }
+
+    public String getRoutingKey() {
+        return routingKey;
+    }
```

Comment:
We can use `@RequiredArgsConstructor` and `@Getter` annotation for class instead

Reply:
Getters generated by Lombok aren't visibile to Kotlin compiler unless Kotlin code is in a different module

Reply:
Ok, I get it. Maybe we should convert that class to a Kotlin data class?

Reply:
Yeah, we probably should